disconnect-blacklist.json

```diff
@@ -8963,10 +9304,11 @@
             "tweetdeck.com",
             "twimg.com",
             "twitter.com",
-            "twitter.jp"
+            "twitter.jp",
+            "ads-twitter.com"
```

Comment:
Dang, this causes `[ERROR] ads-twitter.com not found in disconnect_mapping` in the `lists2safebrowsing.py` script. I filed https://github.com/mozilla-services/shavar-list-creation/issues/63 to fix this up.